components/dashboard/src/dedicated-setup/SetupLayout.tsx

```diff
@@ -55,7 +55,7 @@ export const SetupLayout: FC<Props> = ({
                         )}
                     </div>
                 </div>
-                <div className={`mt-24 ${noMaxWidth ? "" : "max-w-md"}`}>
+                <div className={`mt-24 ${noMaxWidth ? "" : "max-w-md"} pb-4`}>
```

Comment:
this drive-by fixes the spacing on the bottom of the setup page, where the "Verify SSO configuration" button had no space below it.